go.mod

```
@@ -5,8 +5,8 @@ go 1.13
 require (
 	github.com/google/goexpect v0.0.0-20200703111054-623d5ca06f56 // indirect
 	github.com/google/goterm v0.0.0-20190703233501-fc88cf888a3f // indirect
-	github.com/u-root/u-root v6.0.0+incompatible
-	golang.org/x/crypto v0.0.0-20191002192127-34f69633bfdc
+	github.com/u-root/u-root v6.0.1-0.20200822172051-c6181ee270c5+incompatible
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	google.golang.org/grpc v1.30.0 // indirect
 	harvey-os.org v0.0.0-20200727215830-5b0d35efdc89 // indirect
```

This version of `harvey-os.org` differs from the one in `bootstrap.sh`.
```suggestion
	harvey-os.org v0.0.0-20200818173457-bcfa7228c80c // indirect
```